kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/api/v1/dto/BatchRequest.java

```diff
@@ -90,9 +87,6 @@ public void setName(String name) {
   }
 
   public Map<String, String> getConf() {
-    if (null == conf) {
-      return Collections.emptyMap();
-    }
```

Comment:
I mean keep the null check but change the default value, otherwise, it may introduce regression if user calls `setConf(null)` explicitly before calling `getConf`

Reply:
SGTM.